src/hooks/useAnalytics.ts

```diff
@@ -50,6 +50,12 @@ export const useAnalytics = () => {
             ? 'DESKTOP_LARGE'
             : 'UNSUPPORTED';
 
+  // AnalyticsUserProperty.Geo
+  const geo = useAppSelector(getGeo) ?? undefined;
```

Comment:
tiny nit: sorta weird we're using `?? undefined` and then `?? null` below; do we need the `?? undefined` here? I think `getAppSelector(getGeo)` should just return a `Nullable<string>` anyways so maybe `identify(AnalyticsUserProperties.Geo(geo)` will just work as expected?

Reply:
😢  identify(AnalyticsUserProperties.Geo(geo) doesn't work which si why we did this. the Nullable<string> type is a little weird.  happy to sychronize the 2 though and just make this `?? null` though!

Reply:
ah man, i forgot to push updates 😢 i'm sorry! will fix in a followup PR soon